src/styles/partials/components/_breadcrumbs-dark.scss

```diff
@@ -1,19 +1,24 @@
 .dark,
 .dg_page-header.blue {
   .dg_breadcrumbs {
-    color: $gold;
+    .breadcrumb {
+      color: $white;
 
-    li.dg_breadcrumb {
-      a {
+      &:hover {
+        color: $blue-lightest;
+      }
+
+      &:after {
         color: $white;
+      }
+
+      &.breadcrumb-last {
```

Comment:
would :last-child work here instead?

Reply:
I think they would do the same thing, so I'm not committed to either option since I'm not seeing many negatives on either option. I will say, SE will always add the class `.breadcrumb-last` to the last child within the breadcrumbs module, so I think targeting `.breadcrumb-last` is catering more towards SE's intended usage of the styling. Let me know which method you would prefer, with that in mind. If we do end up using the `.breadcrumb-last` class as a parent, looking at it again, I should probably move the margin style under that as well.

Reply:
Migrated all children of `.breadcrumb-last` to `:last-child`.

Reply:
Applied scss changes to _breadcrumbs-dark.scss as well.